clash/bloody_grail.tex

```diff
@@ -0,0 +1,129 @@
+    \item Whenever an Astrologers Proclaim or Event card that allows you to Recruit Neutral Units is drawn, ignore it, and draw a new Astrologers Proclaim or Event card.
+    \item Players can use their Deck of Might \& Magic when paying gold to defend their faction Town.
+    \item Ignore Combat encounters on the field with the Grail.
+    \item Players may not Visit the field with the Grail token unless they have already Visited at least two Obelisks or the Grail token has been taken by any Hero at least once.
+    \item To obtain the Grail token, a player’s Hero must spend 2 MPs on the field with the Grail.
+    \item If a player does not possess the title of ``\textcolor{darkcandyapplered}{Blood Knight},'' they must Remove a faction Unit card from their Unit deck immediately after obtaining the Grail token. This effect can only occur once per player.
+    \item If another Hero defeats the Hero with the Grail token, they also take the Grail token.
+    \item If a Hero with the Grail token Surrenders, the Grail token is placed on the hex where the Hero Surrendered.
+    \item If a Neutral army defeats the Hero with the Grail token, the Grail token is placed on the field where the Hero was defeated.
+    \item If a Hero with the Grail token casts Town Portal or uses the Inferno faction Castle Gate building, the Grail token is placed on the hex where the spell was cast or the building used.
+    \item The Grail token behaves like a Permanent card that does not count towards your hand limit and cannot be discarded; it grants the following effect when played: ``$\infty$ Once during each Combat, if the Hero is a \textcolor{darkcandyapplered}{Blood Knight}, discard a card to remove up to 2 \svg{damage-red} from your units and assign it to enemy units.''
+    \item A Hero with the Grail token who possesses the title of ``\textcolor{darkcerulean}{Grail Knight}'' may pay 10 \svg{gold} or 3 \svg{valuables} at any Temple map location to cleanse the Grail. Place a blue faction cube on the Grail token to mark this effect.
+    \item Once cleansed, the Grail token behaves like a Permanent card that does not count towards your hand limit and cannot be discarded; it grants the following effect when played: ``$\infty$ Once during each Combat round, if the Hero is a \textcolor{darkcerulean}{Grail Knight}, discard a card to remove up to 2 \svg{damage-red} from your units.''
+\end{itemize}
+
+\subsection*{\MakeUppercase{Victory Condition}}
+To win the scenario, a Hero must obtain the Grail token and bring it to their faction Town.
+
+\subsection*{\MakeUppercase{Round Tracker}}
+If the Grail is not obtained at least once by the end of the 13th round, the game ends, and all players lose the scenario.
+
+If the Grail token has been obtained, the players gain additional time – until the end of the 16th round – to bring the Grail to their faction Town. Otherwise, all players lose the scenario.
```

Comment:
```suggestion
\subsection*{\MakeUppercase{Lose Conditions}}
If the Grail is not obtained at least once by the end of the 13th round, the game ends, and all players lose the scenario.

If the Grail token has been obtained, the players gain additional time – until the end of the 16th round – to bring the Grail to their faction Town. Otherwise, all players lose the scenario.
```

Reply:
Or you can separate this "Round Tracker" and "Defeat Conditions"?